packages/cli/test/unit/commands/integration/add.test.ts

```diff
@@ -333,7 +339,8 @@ describe('integration', () => {
         it('should error when no integration arugment was passed', async () => {
           client.setArgv('integration', 'add');
           const exitCodePromise = integrationCommand(client);
```

Comment:
Do we want to move these cases where we directly `await` the promise into a single `const exitCode = await <command>;` instances?